public/static/docs/command-reference/update.md

```diff
@@ -69,3 +81,41 @@ stable.
 > Note that `dvc update` updates the `rev_lock` field of the corresponding
 > [DVC-file](/doc/user-guide/dvc-file-format) (when there are changes to bring
 > in).
+
+## Example: Updating imported artifacts to a specified revision
+
+Let's import a data artifact from an older commit from our
+[get started example repo](https://github.com/iterative/example-get-started) at
+first:
+
+```dvc
+$ dvc import --rev baseline-experiment git@github.com:iterative/example-get-started model.pkl
+Importing 'model.pkl (git@github.com:iterative/example-get-started)'
+-> 'model.pkl'
+```
+
+After this, the import stage (DVC-file) `model.pkl.dvc` is created.
+
+Let's try to run `dvc update` on the given stage file, and see what happens.
+
+```dvc
+$ dvc update model.pkl.dvc
+```
+
+There was no output at all, meaning, the `model.pkl` file was not updated. This
+is because, we tied the import stage with a `rev` that never changes (i.e. tag
+is tied to a specific commit). Therefore, it was not updated.
+
+Let's try to update the model to a different experiment `bigrams-experiment`:
+
+```dvc
+$ dvc update --rev bigrams-experiment model.pkl.dvc
+Importing 'model.pkl (git@github.com:iterative/example-get-started)'
+-> 'model.pkl'
```

Comment:
Please remove.

Reply:
Why?

Reply:
@jorgeorpinel, I wanted to show the outputs here. If there's no output, the data is not updated.

Reply:
> Why?

Our style from what I remember is to not include output unless there's a good reason and I thought there was no reason. Up to you

> If there's no output, the data is not updated.

On this (out of scope) should `update` print a message when it can't update import stages because they're fixed to a revision, instead of just retuning quietly?

Reply:
Good point. I'll create a issue for that. We could discuss there.

Reply:
Thanks. Resolving this one then but if you can link here to the issue that would be great.

> And again @skshetry if you have lots of other work I can take this review over since it the PR is already merged, np just lmk so I don't loose track 🙂